scripts/upgrade/upgrade_rewards.py

```diff
@@ -0,0 +1,52 @@
+"""
+  Script to pass governance rewards to multisig
+"""
+from rich.console import Console
+
+from config.badger_config import badger_config
+from scripts.systems.badger_system import BadgerSystem, connect_badger
+from helpers.time_utils import days, to_days, to_utc_date
+from eth_abi import encode_abi
+from brownie import *
+
+
+console = Console()
+
+## Change these to customize functionality
+CONTROLLER_ADDRESS = "0x63cF44B2548e4493Fd099222A1eC79F3344D9682"
+GOVERNANCE_MULTISIG_ADDRESS = "0xB65cef03b9B89f99517643226d76e286ee999e77"
+FUNCTION_TO_CALL = "setRewards(address)"
+
+def test_update_rewards(badger: BadgerSystem) -> str:
```

Comment:
Looks solid to me, I don't think we need anything too fancy for these scripts. I think a generic version of these functions would be great. (pass in all parameters and a function to call with post-run logic to check -> execute the upgrade flow -> run post-checks)